### DISCUSSION TITLE
Peça seu code review aqui!

### DISCUSSION THREAD

**Discussion by romulosardinha** [frontendbr/forum#1733] [Mostre seu trabalho]
Percebo que uma das forma mais eficientes de aprendermos a ser bons profissionais é colocando a mão na massa! Escrevendo código e nos colocando a prova, ou no melhor dos casos, recebendo review de outros.

Então proponho isso. Alguém manda o link de um código no github e especifica qual tecnologia e qual o objetivo do código. E quem quiser avalia o código, dá dicas de melhoras, onde acertou, onde errou... etc...

Talvez seja uma forma boa de organizar: ( por favor, deem opiniões sobre como podemos melhorar essa padronização)

[ OBJETIVO DO CÓDIGO ]
[ TECNOLOGIAS USADAS ]
[ LINK DO CÓDIGO ]

Em exemplo que eu posso dar é um caso que eu mesmo fiz aqui, #1638 

Authentication
js, React-native, contex api, firebase
https://github.com/romulosardinha/login-react-firebase

Quando fiz esta issue, tive otimas dicas e ajudas desta comunidade. Inclusive agradecendo o @ninetails pela ajuda!

Espero que o máximo de pessoas possam ajudar outros =)

**Comment by felipe-heredia** [comment]
### Analise o meu projeto.

Um bot para discord.
_discord.js e mongodb._
[Beam Bot](https://github.com/felipesuri/beam-bot)

**Comment by theandersonn** [comment]
@romulosardinha Bem legal a ideia, poderia inclusive ser uma nova seção aqui no grupo.
Sugiro enviar a proposta aqui: https://github.com/frontendbr/sugestoes

**Comment by renato29** [comment]
Ola!
poderiam ver meu codigo,estou comecando no JS

criar condicional simples
JS
https://github.com/renato29/site-de-soma/blob/master/condicional_simples_JS

**Comment by portothree** [comment]
> Ola!
> poderiam ver meu codigo,estou comecando no JS
> 
> criar condicional simples
> JS
> https://github.com/renato29/site-de-soma/blob/master/condicional_simples_JS

Seria interessante se você implementar um formulario no html e sua função fosse chamada no evento de 'submit', assim pode deixar o button com o type padrão de 'submit' e colocar labels nos seus inputs pra ter uma melhor semântica geral do que o seu snippet faz.

```html
<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
        <title>Detran</title>
    </head>
    <body>
        <form onsubmit="calcular()">
            <h1>Sistema de Multas</h1>
            <div>
                <label for="txtvel">Velocidade do carro:</label>
                <input type="number" name="txtvel" id="txtvel" /> Km/h
            </div>
            <button>Verificar</button>
        </form>
        <div id="res"></div>
        <script>
            function calcular() {
                var txtv = window.document.querySelector("input#txtvel");
                var res = window.document.querySelector("div#res");
                var vel = Number(txtv.value);
                res.innerHTML = `Sua velocidade é de ${vel}Km/h.`;
            }
        </script>
    </body>
</html>
```

**Comment by renato29** [comment]
po mt obrigado! deu mega moral, achei o erro aqui !

**Comment by ninetails** [comment]
Opa muito _massa_ a iniciativa :D

@romulosardinha, sabe que _tamojunto_ se rolar uehauhaua e estamos ae pra isso!

São poucos os lugares para discussões de arquiteturas e boas práticas pra frontend. DIgo, sinto um pouco de falta de discussões do tipo.

**Comment by romulosardinha** [comment]
Vlw @ninetails obrigado cara. Fiz uma sugestão no repositório para ver se tem uma forma de gerar um local focado nisso.
